src/arch/esp32-s3.h

```diff
@@ -279,12 +283,6 @@ void _PM_timerInit(Protomatter_core *core) {
   desc.buffer = core->screenData;
   desc.next = NULL;
 
-  // Alloc DMA channel & connect it to LCD periph
-  gdma_channel_alloc_config_t dma_chan_config = {
-      .sibling_chan = NULL,
-      .direction = GDMA_CHANNEL_DIRECTION_TX,
-      .flags = {.reserve_sibling = 0}};
-  esp_err_t ret = gdma_new_channel(&dma_chan_config, &dma_chan);
```

Comment:
Not sure why this was removed?

Reply:
These variables are initialized but left unutilized.

Reply:
Good Catch! I only meant to remove the unutilized `esp_err_t ret` part.